mlflow/genai/git_versioning/git_info.py

```diff
@@ -43,3 +49,10 @@ def from_env(cls) -> Self:
 
         except git.GitError as e:
             raise GitOperationError(f"Failed to get repository information: {e}") from e
+
+    def to_mlflow_tags(self) -> dict[str, str]:
+        return {
+            MLFLOW_GIT_VERSIONING_BRANCH: self.branch,
+            MLFLOW_GIT_VERSIONING_COMMIT: self.commit,
```

Comment:
Do we already have existing tags for git branch / commit? If so, can we use them? Also, can we include git repo as a tag?

Reply:
I'll file a separate PR for this.

Reply:
Filed https://github.com/mlflow/mlflow/pull/17051